src/client/services/MCFService.js

```diff
@@ -56,24 +56,12 @@ export class MCFService {
       })
   }
 
-  getHeatMap = genes => {
+  getHeatMap = (genes, type) => {
```

Comment:
should I rename genes to something else, since for the intronAnalysisHeatMap its only a singular gene

Reply:
`genes` is fine.